hack/multi-node/user-data.sample

```diff
@@ -32,8 +30,8 @@ coreos:
         ExecStart=/usr/lib/coreos/kubelet-wrapper \
           --runonce \
           --runonce-timeout=60s \
-          --api-servers=https://172.17.4.101:6443 \
-          --kubeconfig=/etc/kubernetes/kubeconfig.yaml \
+          --api-servers=https://172.17.4.101:443 \
```

Comment:
The 6443 was due to not being able to (easily) bind to a port <1024. But it's somewhat non-standard. Users will likely expect to be able to hi "https://kubernetes" for the api-server (and not rely on needing to determine port as well)